Build failure: qtconnectivity on darwin

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

install package normcap. qtconnectivity is a dependency.


### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

TODO

### Relevant log output

```console
error: Cannot build '/nix/store/fdyc715iv4fmwldwxxj3p9k7mi2542yc-qtconnectivity-6.10.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/ckc99smz7nzyv1x4skqg64gfmgnsxi9d-qtconnectivity-6.10.0-dev
         /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0
       Last 25 log lines:
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/CQNNGYK7HN/../../../../../src/nfc/pcsc/qpcsccard_p.h:18:
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/CQNNGYK7HN/../../../../../src/nfc/pcsc/qpcsc_p.h:23:
       > /nix/store/sdlxwfdsp122iiik7113k7dah7ism1ji-pcsclite-2.3.0-dev/include/PCSC/winscard.h:41:10: error: 'pcsclite.h' file not found with <angled> include; use "quotes" instead
       >    41 | #include <pcsclite.h>
       >       |          ^
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/mocs_compilation.cpp:2:
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/CQNNGYK7HN/moc_qpcsccard_p.cpp:9:
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/CQNNGYK7HN/../../../../../src/nfc/pcsc/qpcsccard_p.h:18:
       > In file included from /nix/store/ma8srjjmxq7mhbphv9n1vmkk9xc4y2vh-qtconnectivity-6.10.0/.build/qtconnectivity-everywhere-src-6.10.0/build/src/nfc/Nfc_autogen/CQNNGYK7HN/../../../../../src/nfc/pcsc/qpcsc_p.h:23:
       > In file included from /nix/store/sdlxwfdsp122iiik7113k7dah7ism1ji-pcsclite-2.3.0-dev/include/PCSC/winscard.h:41:
       > /nix/store/sdlxwfdsp122iiik7113k7dah7ism1ji-pcsclite-2.3.0-dev/include/PCSC/pcsclite.h:45:10: error: 'wintypes.h' file not found with <angled> include; use "quotes" instead
       >    45 | #include <wintypes.h>
       >       |          ^
       > 2 errors generated.
       > [29/94] Building CXX object src/nfc/CMakeFiles/Nfc.dir/qndeffilter.cpp.o
       > [30/94] Building CXX object src/nfc/CMakeFiles/Nfc.dir/qnearfieldmanager.cpp.o
       > [31/94] Building CXX object src/nfc/CMakeFiles/Nfc.dir/qndefnfcsmartposterrecord.cpp.o
       > [32/94] Building CXX object src/nfc/CMakeFiles/Nfc.dir/qnearfieldtarget_p.cpp.o
       > [33/94] Building CXX object src/nfc/CMakeFiles/Nfc.dir/qnearfieldtarget.cpp.o
       > [34/94] Building CXX object src/bluetooth/CMakeFiles/Bluetooth.dir/Bluetooth_autogen/mocs_compilation.cpp.o
       > [35/94] Building CXX object src/bluetooth/CMakeFiles/Bluetooth.dir/removed_api.cpp.o
       > [36/94] Building CXX object src/bluetooth/CMakeFiles/Bluetooth.dir/qbluetooth.cpp.o
       > [37/94] Building CXX object src/bluetooth/CMakeFiles/Bluetooth.dir/qbluetoothaddress.cpp.o
       > [38/94] Building CXX object src/bluetooth/CMakeFiles/Bluetooth.dir/qbluetoothdevicediscoveryagent.cpp.o
       > ninja: build stopped: subcommand failed.
       For full logs, run:
         nix log /nix/store/fdyc715iv4fmwldwxxj3p9k7mi2542yc-qtconnectivity-6.10.0.drv
```

### Additional context

_No response_

### System metadata

 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.6.0, macOS 15.7.2`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/7yzwp99zg2qsv3pwf3b4jariqxfid8n7-source`

### Notify maintainers

@NickCao



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/qt-6/modules/qtconnectivity.nix
+++ b/pkgs/development/libraries/qt-6/modules/qtconnectivity.nix
@@ -12,7 +12,10 @@
 qtModule {
   pname = "qtconnectivity";
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ pcsclite ] ++ lib.optionals stdenv.hostPlatform.isLinux [ bluez ];
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
+    pcsclite
+    bluez
+  ];
   propagatedBuildInputs = [
     qtbase
     qtdeclarative

